js/company.js

```diff
@@ -0,0 +1,175 @@
+const SECONDARY_COLOR = "#0f9404";
+const CHART_WIDTH = 800;
+const CHART_HEIGHT = 500;
+var urlParams = new URLSearchParams(window.location.search);
+
+let symbol = urlParams.get("symbol");
+
+document.title = SYMBOL_PREFIX + symbol + " - Company Info";
+
+showLoadingCompany();
+
+fetch(`${STOCKS_BASE_URL}/api/v3/company/profile/${symbol}`)
```

Comment:
consider using async await syntax, it will make you code much easier to read and understand.